### PR TITLE
Page: Tina Cloud updates

### PR DESCRIPTION
**Pull request by DirtyF**
- Text copy changes. 
- Branching is not available yet on Tina Cloud

### CODE REVIEW

**Review by jamespohalloran — CHANGES_REQUESTED** [review]

File: content/pages/cloud.json [hunk 1]
```diff
@@ -51,27 +51,27 @@
       "items": [
         {
           "_template": "feature",
-          "headline": "Collaborate With the Team",
-          "subline": "Onboard your team; no GitHub accounts required.  Tina Cloud brings user-management, auth, roles, and a GraphQL API for your content.",
+          "headline": "Collaborate as a Team",
+          "subline": "Tina Cloud brings authentication, user management, roles. Onboard your team in no time, no GitHub accounts required.",
```
Comment (DirtyF) [inline content/pages/cloud.json:55]:
```suggestion
          "subline": "Tina Cloud brings authentication, user management, roles. Onboard your team in no time, no GitHub accounts for editors required.",
```

File: content/pages/cloud.json [hunk 2]
```diff
@@ -51,27 +51,27 @@
       "items": [
         {
           "_template": "feature",
-          "headline": "Collaborate With the Team",
-          "subline": "Onboard your team; no GitHub accounts required.  Tina Cloud brings user-management, auth, roles, and a GraphQL API for your content.",
+          "headline": "Collaborate as a Team",
+          "subline": "Tina Cloud brings authentication, user management, roles. Onboard your team in no time, no GitHub accounts for editors required.",
```
Comment (DirtyF) [inline content/pages/cloud.json:55]: @jamespohalloran fixed :)

File: content/pages/cloud.json [hunk 3]
```diff
@@ -51,27 +51,27 @@
...
         {
           "_template": "feature",
-          "headline": "Move Faster With Tina Cloud",
-          "subline": "Visual editing + Git storage allows you to ship content faster with the safety of version-control.",
+          "headline": "Ship Content Faster",
           "media": {
             "src": "/img/visual-editing.gif"
-          }
+          },
+          "subline": "Visual editing greatly improves the productivity of content teams and make their job much more enjoyable. You team can work with full confidence, powerful Git Version Control has your back."
```
Comment (jamespohalloran) [inline content/pages/cloud.json:66]:
```suggestion
          "subline": "Visual editing greatly improves the productivity of content teams and makes their job much more enjoyable. Your team can work with full confidence. Powerful Git version control has your back."
```

File: content/pages/cloud.json [hunk 4]
```diff
@@ -51,27 +51,27 @@
...
         {
           "_template": "feature",
-          "headline": "GraphQL API",
-          "subline": "The Tina Content API provides a GraphQL interface to your repositories files where content is stored in open formats like Markdown and JSON, allowing for easy content re-use and migrations.",
+          "headline": "Keep your Data in Check",
           "media": {
             "src": "/img/graphQL.png"
-          }
+          },
+          "subline": "Tina Cloud Content API provides a GraphQL interface to your repositories files to bring data integrity and relationships to your filesystem.\nStore your content in open formats like Markdown and JSON to ease content re-use and migrations."
```
Comment (jamespohalloran) [inline content/pages/cloud.json:74]:
```suggestion
          "subline": "Tina Cloud Content API provides a GraphQL interface to your repositories' files, bringing data integrity and relationships to your filesystem.\nStore your content in open formats like Markdown and JSON to ease content re-use and migrations."
```

> bringing data integrity and relationships to your filesystem.

"Filesystem" throws me off here. Might just be me, but maybe this is clearer?:

> bringing data integrity and relationships to your files' content.

(Github has a GraphQL Api for files, but our values comes from querying the content within them).

**Review by jamespohalloran — CHANGES_REQUESTED** [review]

File: content/pages/cloud.json [hunk 5]
```diff
@@ -51,27 +51,27 @@
...
         {
           "_template": "feature",
-          "headline": "GraphQL API",
-          "subline": "The Tina Content API provides a GraphQL interface to your repositories files where content is stored in open formats like Markdown and JSON, allowing for easy content re-use and migrations.",
+          "headline": "Keep your Data in Check",
           "media": {
             "src": "/img/graphQL.png"
-          }
+          },
+          "subline": "Tina Cloud Content API provides a GraphQL interface to your repositories' files, bringing data integrity and relationships to your filesystem.\nStore your content in open formats like Markdown and JSON to ease content re-use and migrations."
```
Comment (DirtyF) [inline content/pages/cloud.json:74]:
```suggestion
          "subline": "Tina Cloud Content API provides a GraphQL interface to your repositories' files, bringing data integrity and relationships to your content files.\nStore your content in open formats like Markdown and JSON to ease content re-use and migrations."
```

**Pull request closed without merging**